test/boolattrs.ts

```diff
@@ -0,0 +1,32 @@
+"use strict";
```

Comment:
What's the need of `use strict` with TypeScript? Isn't strict mode already enabled in tsconfig.json?